editor/src/components/navigator/navigator.spec.browser2.tsx

```diff
@@ -4757,8 +4757,8 @@ describe('Navigator', () => {
           'regular-utopia-storyboard-uid/scene-aaa/app-entity:root/new-container',
           'regular-utopia-storyboard-uid/scene-aaa/app-entity:root/old-container',
           'regular-utopia-storyboard-uid/scene-aaa/app-entity:root/old-container/wrapper',
-          'regular-utopia-storyboard-uid/dragme1',
           'regular-utopia-storyboard-uid/dragme2',
+          'regular-utopia-storyboard-uid/dragme1',
```

Comment:
I have no idea why the order of the derived targets has changed here, but I've tested this interaction on this project in the editor and the navigator (and resulting code) was ordered correctly...